package.json

```diff
@@ -0,0 +1,117 @@
+{
+  "name": "gatsby-vscode",
+  "version": "1.0.0",
+  "description": "Gatsby VSCode extension for GraphQL support and more",
+  "publisher": "Gatsby",
+  "displayName": "Gatsby VSCode",
+  "main": "dist/extension",
+  "engines": {
+    "vscode": "^1.8.0"
+  },
+  "preview": true,
+  "extensionDependencies": [
+    "graphql.vscode-graphql"
```

Comment:
this is where we install our `vscode-graphql` extension, where most of the graphql logic lives